vllm_hpu_extension/test_capabilities.py

```diff
@@ -0,0 +1,97 @@
+###############################################################################
+# Copyright (C) 2024 Habana Labs, Ltd. an Intel Company
+#
+# This source code is licensed under the Apache 2.0 license found in the
+# LICENSE file in the root directory of this source tree.
+###############################################################################
+
+import pytest
+
+from vllm_hpu_extension.capabilities import VersionRange, Capabilities
+
+
+def test_operators():
+    assert VersionRange("<1.19.0.100")(build="1.19.0.99")
```

Comment:
you use different version notation in capabilities.py ("index_copy": VersionRange(">=1.19.0-272"),) - dash instead of last dot

Reply:
Thanks for catching that!

Reply:
We should use python notation - everything with dots as the '-' has a bit different meaning